components/formats-gpl/src/loci/formats/in/CellWorxReader.java

```diff
@@ -461,7 +461,51 @@ else if (seriesIndex < seriesCount - 1) {
 
     MetadataStore store = makeFilterMetadata();
     MetadataConverter.convertMetadata(convertMetadata, store);
-    MetadataTools.populatePixels(store, this);
+    MetadataTools.populatePixels(store, this, true);
+
+    // check for stage positions in each file
+
+    for (int s=0; s<getSeriesCount(); s++) {
```

Comment:
Both sub-readers (`DeltavisionReader` and `MetamorphReader`) will skip the plane position metadata population if a minimum metadata level is passed. It might useful to add a check for the metadata level before this block. This may slightly reduce the performance overhead when calling e.g. `showinf -nometa`.

Reply:
Should be fixed by bee72b4.